llama_index/indices/base.py

```diff
@@ -244,7 +244,7 @@ def delete_ref_doc(
             return
 
         self.delete_nodes(
-            ref_doc_info.node_ids,
+            [ref_doc_id],
```

Comment:
This will break a lot of other indexes I think.

What if we just delete both `ref_doc_info.node_ids + [ref_doc_id]` ?